include/igl/opengl/glfw/Viewer.cpp

```diff
@@ -123,16 +123,16 @@ namespace opengl
 namespace glfw
 {
 
-  IGL_INLINE int Viewer::launch(bool resizable,bool fullscreen)
+  IGL_INLINE int Viewer::launch(bool resizable /*= true*/,bool fullscreen /*= false*/, int windowWidth /*= 1280*/, int windowHeight /*= 800*/)
   {
     // TODO return values are being ignored...
-    launch_init(resizable,fullscreen);
+    launch_init(resizable,fullscreen,windowWidth,windowHeight);
     launch_rendering(true);
     launch_shut();
     return EXIT_SUCCESS;
   }
 
-  IGL_INLINE int  Viewer::launch_init(bool resizable,bool fullscreen)
+  IGL_INLINE int  Viewer::launch_init(bool resizable,bool fullscreen, int windowWidth, int windowHeight)
   {
```

Comment:
propagated here

Reply:
I am getting a strange behaviour when dragging outside for rotating and for moving

Reply:
Dragging on another viewport is still buggy, I suspect might the same issue when dragging outside.

Reply:
the dragging outside keeps the last view port selected
for the moving, i feel that is a problem of mouse offset, when you move to the new viewport it keeps the delta and the second object "jumps"